### ISSUE TITLE
Weird terminal output after upgrading to jupyter notebook v5

### ISSUE THREAD

**Issue by ajones27** [ipython/ipython#10474]
Hi, when I upgraded the notebook (using conda) I got a weird output on the terminal, even when opening the notebook with the link provided. Is this a common problem? It pings my terminal 5 or 6 times every time I open the notebook:

<img width="1032" alt="screen shot 2017-04-21 at 12 03 10" src="https://cloud.githubusercontent.com/assets/15875811/25273122/b718daa2-268a-11e7-898d-bb74d350f67a.png">

**Comment by takluyver** [comment]
That's really weird. It looks like a debugging print statement has got left in somewhere, but it's not happening for me. Do you have any custom configuration or extensions installed?

**Comment by ajones27** [comment]
I don't think so, I haven't installed anything special. i can always post my requirements file if it'll help? in case there's something else that might be interfering with it

**Comment by takluyver** [comment]
I can't currently think where to begin debugging this, unfortunately.

**Comment by ajones27** [comment]
I tried removing the notebook and upgrading again. 

1) when i did 'pip install --upgrade notebook" i get:

> Cannot remove entries from nonexistent file $HOME/.conda/envs/py27/lib/python2.7/site-packages/easy-install.pth

2) When i try with conda install, it seems to work fine until i run 'jupyter notebook', at which point i get:


> File "$HOME/.conda/envs/py27/bin/jupyter-notebook", line 4, in <module>
>     import notebook.notebookapp
>   File "$HOME/.conda/envs/py27/lib/python2.7/site-packages/notebook/notebookapp.py", line 62, in <module>
>     from .services.contents.filemanager import FileContentsManager
>   File "$HOME/.conda/envs/py27/lib/python2.7/site-packages/notebook/services/contents/filemanager.py", line 15, in <module>
>     from .filecheckpoints import FileCheckpoints
>   File "$HOME/.conda/envs/py27/lib/python2.7/site-packages/notebook/services/contents/filecheckpoints.py", line 15, in <module>
>     from . import tz

**Comment by takluyver** [comment]
Is that all of the traceback? It looks like it's cut off before the error.

**Comment by ajones27** [comment]
Sorry, you're right, the last line was "ImportError: cannot import name tz"

**Comment by takluyver** [comment]
Hmm, I think it's probably a clash between conda and pip trying to manage the same package. Try to uninstall everything related to Jupyter from both packaging systems, then run `conda clean -tipsy` to clear out its cached data, and then reinstall Jupyter with just one packaging system.

**Comment by ajones27** [comment]
I tried just now (pip uninstall jupyter/notebook/ipython, conda uninstall jupyter/notebook/ipython, conda clean -tipsy, pip install jupyter) but had the same problem. is there anything else related to jupyter that i'm not uninstalling?

**Comment by takluyver** [comment]
Are you able to remove Anaconda entirely and the whole `.conda` directory containing your envs and then reinstall it? It shouldn't be necessary, but working out how to fix it without doing that may be pretty awkward.
